pub-post-iati-identifiers-exist/index.js

```diff
@@ -0,0 +1,132 @@
+import config from '../config/config.js';
+import { query } from '../solr/solr.js';
+
+async function testIatiIdentifiers(queryUrl, identifiers) {
+    const iatiIdentifiersQuery = identifiers.map(identifier => 
+                                                    `iati_identifier_exact:${JSON.stringify(identifier)}`).join(" ")
+
+    const requestBody = {
+        query: iatiIdentifiersQuery,
+        params: {rows: 0},
+        facet: {
+            "identifiers" : {
+                    type: "terms",
+                    field : "iati_identifier",
+                    offset: 0,
+                    limit: 1000
```

Comment:
Should this 1000 in some way be passed in from the calling function? Do we need to make sure it's bigger than BATCH_SIZE?

Reply:
Ah yes, it should just be BATCH_SIZE. Will redo.